src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -3927,6 +3927,23 @@ internal uint GetValEscape(BoundExpression expr, uint scopeOfTheContainingExpres
                             isRefEscape: false);
                     }
 
+                    if (conversion.Conversion.IsUserDefined)
+                    {
+                        var operatorMethod = conversion.Conversion.Method;
+                        Debug.Assert(operatorMethod is not null);
+
+                        return GetInvocationEscapeScope(
+                            operatorMethod,
+                            receiver: null,
+                            receiverIsSubjectToCloning: ThreeState.Unknown,
+                            operatorMethod.Parameters,
+                            argsOpt: [conversion.Operand],
```

Comment:
>conversion.Operand [](http://example.com/codeflow?start=38&length=18)

Is this going to include a possible standard implicit conversion from original argument to the parameter type?

Reply:
Yes, same what would happen for normal call.